src/Server/RestRouterHandlers/TableRestRouterHandler.cpp

```diff
@@ -112,6 +114,13 @@ String TableRestRouterHandler::executeGet(const Poco::JSON::Object::Ptr & /* pay
 
 String TableRestRouterHandler::executePost(const Poco::JSON::Object::Ptr & payload, Int32 & /*http_status*/) const
 {
+    const auto & database_name = getPathParameter("database");
```

Comment:
do we need update `http_status` in this function ?

Reply:
fixed